src/imitation/util/logger.py

```diff
@@ -3,23 +3,70 @@
 import contextlib
 import datetime
 import os
+import sys
 import tempfile
 from typing import Any, Dict, Generator, Optional, Sequence, Tuple, Union
 
-import stable_baselines3.common.logger as sb_logger
+from stable_baselines3.common.logger import (
```

Comment:
Style guide violation, we only import modules not classes (except for `typing`): https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Thanks! I wasn't aware of this one.